src/items.rs

```diff
@@ -408,6 +408,122 @@ impl<'a> FmtVisitor<'a> {
         }
     }
 
+    pub fn visit_enum(&mut self,
+                      ident: ast::Ident,
+                      vis: ast::Visibility,
+                      enum_def: &ast::EnumDef,
+                      generics: &ast::Generics,
+                      span: Span)
+    {
+        // Empty enums are a bit weird (and not like empty structs), let's not touch them for now
```

Comment:
You are correct. I was afraid of dropping comments, but they can easily be preserved.